src/AspNet.Security.OAuth.EVEOnline/EVEOnlineAuthenticationExtensions.cs

```diff
@@ -4,7 +4,12 @@
  * for more information concerning the license and the contributors participating to this project.
  */
 
+using System;
+using System.IdentityModel.Tokens.Jwt;
 using AspNet.Security.OAuth.EVEOnline;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection.Extensions;
```

Comment:
I feel some of these are probably redundant now we're using implicit using statements in C# 10.